scripts/run_lighthouse_tests.py

```
@@ -169,52 +156,142 @@ def get_entity(line: str) -> tuple[str, str] | None:
+def _run_lighthouse_checks_for_config(config_filename: str) -> None:
+    """Runs the Lighthouse checks through the given Lighthouse config."""
     lhci_path = os.path.join('node_modules', '@lhci', 'cli', 'src', 'cli.js')
     # The max-old-space-size is a quick fix for node running out of heap memory
     # when executing the performance tests: https://stackoverflow.com/a/59572966
     bash_command = [
-        common.NODE_BIN_PATH,
+        common.LIGHTHOUSE_NODE_BIN_PATH,
         lhci_path,
         'autorun',
-        '--config=%s' % LIGHTHOUSE_CONFIG_FILENAMES[lighthouse_mode],
+        '--config=%s' % config_filename,
```

Ruff (0.16.2)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)